src/sentry/tasks/post_process.py

```diff
@@ -277,27 +275,45 @@ def post_process_group(
                     safe_execute(callback, event, futures, _with_transaction=False)
 
             try:
+                from sentry.app import locks
+                from sentry.utils.locking import UnableToAcquireLock
+                from sentry.utils.committers import get_frame_paths
```

Comment:
Why are we moving this all inline?

Reply:
Just moved them out.

But to answer your question, I'd just say I did it because they were local imports and I wanted anything I added to be within the try/catch.